clusters/README.md

```diff
@@ -0,0 +1,7 @@
+# Clusters
+
+{% hint style="info" %}
+This section of the documention intends to document the run-time environment offered as a part of NAIS.
```

Comment:
perhaps more explicit like
`... document the runtime environment (Kubernetes clusters) offered ...`

Reply:
Thanks for the suggestion, fix'd! =)